src/controllers/LogViewerController.php

```diff
@@ -27,10 +32,33 @@ public function index()
 
         $logs = LaravelLogViewer::all();
 
-        return View::make('laravel-log-viewer::log', [
+        return View::make(self::cfgHas('view.log') ? self::cfg('view.log') : 'laravel-log-viewer::log', [
```

Comment:
The `config` helper has an optional second parameter that is used as default if the config key doesn't exist:

`config(self::$configFile . '.view.log', 'laravel-log-viewer::log')`